sanity/lib/sanity.js

```diff
@@ -0,0 +1,80 @@
+import Image from 'next/image';
+import {
+  createClient,
+  createPreviewSubscriptionHook,
+} from 'next-sanity';
+import createImageUrlBuilder from '@sanity/image-url';
+import { PortableText as PortableTextComponent } from '@portabletext/react';
+import { config } from './config';
+import GetImage from './getImage';
```

Comment:
Separate external imports from internal imports by adding an empty line between them. External imports should go at the top, followed by internal.